### ISSUE TITLE
Suggestion: Ability to move poe overlay outside the game screen when using multiple monitors.

### ISSUE THREAD

**Issue by oegeak** [Kyusung4698/PoE-Overlay#804]
So even though this is a brilliant application, it does take up quite a bit of screen real-estate. 

I'm not sure if it's possible but I would really like to be able to move the overlay part to another screen so it's more like a "next to lay"

Also, why isn't it possible to scroll trough your stash pressing control while your mouse is outside your stash but in your inventory for example.

Just to be safe, this is regarding the overwolf version.

Kind regards,
Cageman

**Comment by MalakaJones** [comment]
Yes, having the ingame browser on the second monitor would be awsome.
